package.json

```diff
@@ -21,7 +21,7 @@
     "@angular/router": "17.0.2",
     "@testing-library/angular": "14.4.0",
     "@testing-library/user-event": "14.5.1",
-    "ng-mocks": "14.11.0",
+    "ng-mocks": "14.12.1",
```

Comment:
Oh this brings to my attention that we need to cleanup the dependencies 😅

@testing-library packages needs to move to devDependencies, `ng-mocks` isn't used and can thus be removed.
Do you want to update the PR with this?

Reply:
I can do that after work. It is 11:17 AM in my timezone so you will probably be sleeping by then. Otherwise I wouldn't mind if you want to do that now outside of the PR and close this.

Reply:
There's no rush with it.
Please take your time 🙂